app/views/users/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Edit user</h1>
+
+<%= form_for @user do |f| %>
+  <%= f.label :username, "Username:" %><br />
+  <%= f.text_field :username %>
+  <% @user.errors.full_messages_for(:username).each do |message| %>
+  <div><%= message %></div>
+  <% end %>
+  <br /><br />
+  <%= f.label :email, "Email:" %><br />
+  <%= f.text_field :email, placeholder: "Example@example.com" %>
+  <% @user.errors.full_messages_for(:email).each do |message| %>
+  <div><%= message %></div>
+  <% end %>
+  <br /><br />
+  <%= f.label :password, "Password:" %><br />
+  <%= f.password_field :password %>
+  <% @user.errors.full_messages_for(:password).each do |message| %>
+  <div><%= message %></div>
+  <% end %>
+  <br /><br />
+  <%= f.submit %>
```

Comment:
Use `#form_with` here as it was instructed in the [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms) project.